src/components/shared/pcsGrid/pcsGrid.scss

```diff
@@ -77,7 +79,40 @@ $doublePadding: $baseContentPadding * 2;
 
     .ag-header { border-bottom: 1px solid themed('colorGridRowBorder'); }
 
-    .ag-header-cell { color: themed('colorGridCellText'); }
+    .ag-header-cell {
+      color: themed('colorGridCellText');
+
+      .ag-icon-none {
+        visibility: hidden;
+        opacity: 0;
+        transition: visibility 0s, opacity 0.5s linear;
+        background-image: url(themed('gridSortIcon'));
+      }
+
+      .ag-icon-asc { background-image: url(themed('gridAscIcon')); }
+      .ag-icon-desc { background-image: url(themed('gridDescIcon')); }
+
+      $offset: 30px;
+      $resizeHeight: $header-height - $offset;
+      .ag-header-cell-resize::after {
```

Comment:
Need a line separating vars from styles?  Or should the variables be defined at the top?

Reply:
Updated